classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
+    super(publish_date, id)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  # @return boolean
+  def can_be_archived
+    super || @cover_state = 'bad'
```

Comment:
Please try to refactor this line, as you are assigning 'bad' to @cover_state instead of checking the equality.

Maybe you mean `@cover_state  == 'bad' `